shared/wallets/send-form/footer/container.js

```diff
@@ -8,10 +8,12 @@ import {compose, connect, setDisplayName} from '../../../util/container'
 const mapStateToProps = state => {
   const {isRequest} = state.wallets.building
   return {
-    isRequest,
+    calculating: !!state.wallets.building.amount,
     disabled: !(isRequest
       ? state.wallets.builtRequest.readyToRequest
       : state.wallets.builtPayment.readyToSend),
+    isRequest,
+    waitingKey: Constants.buildPaymentWaitingKey,
```

Comment:
any reason to plumb this through instead of just setting it directly in `footer/index.js`? We only use it from one place

Reply:
@akalin-keybase Fair point!  Looks like we pass it down through the container on all of the other Wallet screens, so I'll keep it this way for now.  Guess I got carried away keeping display components as dumb as possible..